pages/platform/databases/connect-kubernetes-to-managed-mysql/guide.en-asia.md

```diff
@@ -0,0 +1,302 @@
+
+### Step 4: Configure database nodes
+
+You can choose the node model in this step. The initial and maximum numbers of nodes depends on the solution chosen in step 2.
+
+![Order nodes](images/connect-kubernetes-to-managed-mysql04.png){.thumbnail}
+
+Please visit the [capabilities page](../mysql/capabilities/) for detailed information on the hardware resources and other properties of the MySQL database installation.
+
+Take note of the pricing information and click on `Next`{.action} to continue.
+
+### Step 5: Configure your options
+
+You can name your database in this step.
+
+![Configure options](images/connect-kubernetes-to-managed-mysql05.png){.thumbnail}
+
+### Connecting a private network (optional)
+
+For now, the solution is not compatible with private networks.
+
+### Step 6: Summary and confirmation
+
+The final section will display a summary of your order as well as the API equivalent of creating this database instance with the [OVHcloud API](../../../api/first-steps-with-ovh-api/).
+
+![Confirm order](images/connect-kubernetes-to-managed-mysql06.png){.thumbnail}
+
+Within a few minutes your new database service will be deployed. Messages in the OVHcloud Control Panel will inform you when the database is ready to use.
+
+You know your database is ready when cluster status is "Ready", node status is green, and number of users is set to "1 user".
+
+![Control Panel DB ok](images/connect-kubernetes-to-managed-mysql08.png){.thumbnail}
+
+## Whitelist your OVHcloud Managed Kubernetes cluster
+
+> [!warning]
+> For security reasons the default network configuration doesn't allow any incoming connections. 
+> To allow access from your OVHcloud Managed Kubernetes service to the database, cluster nodes IPs have to be whitelisted.
+> Adding kubernetes cluster nodes IPs to the whitelist is not recommended on production environnment, and must only be done for testing.
+> In the near future, we will write a guide on how using vrack, and how to interconnect your different managed services.
+
+
+### Add kubernetes cluster nodes IPs to the DB whitelist
+
+Get your Kubernetes cluster nodes IP addresses from the Compute / Instances menu on the left.
+
+![Nodes IPs list](images/connect-kubernetes-to-managed-mysql09.png){.thumbnail}
+
+Following the related documentation [whitelist the suitable IP addresses](../mongodb/managing-service/#configuring-authorised-ips), add your k8s cluster node IPs to the whitelist
+
+![Db completed setup](images/connect-kubernetes-to-managed-mysql10.png){.thumbnail}
+
+## Test connection from k8s to MySQL db
+
+An easy and quick way to test the connection is to start a MySQL client inside a pod, and use the CLI to connect to the db.
+
+### Start a MySQL client inside a pod
+
+```
+kubectl run mysql-client --image=mysql:8 -it --rm --restart=Never -- /bin/bash
+```
+You are now inside the pod you just created on your cluster, you can simply use the MySQL CLI to connect to your manages MySQL database.
```

Comment:
suggestion:
"You are now inside the pod you just created on your cluster, you can simply use the MySQL CLI to connect to your managed MySQL database."